packages/storybook/src/theme.ts

```diff
@@ -30,6 +30,8 @@ const colors = {
   success: baseColors.green[2],
   warning: baseColors.yellow[2],
   danger: baseColors.red[2],
+  ...baseColors,
+  grey: baseColors.gray,
```

Comment:
css has dual grey/gray so we can probably follow that example.

Reply:
Works for me!